### ISSUE TITLE
Limit part variables & controlling two dimensions 

### ISSUE THREAD

**Issue by eheyder** [KeyboardLogic/KeyboardLogic.SolidEdge.AddIn.ItemCatalog#4]
Question from Kevin:
I was wondering we had talked before about being able to ad limits to the Mother parts files so that they could never be greater than a certain length. The extrusions are only made so long in real life,  but our “ Laenge “ value could be anything that we want to type in .
                Another  example of where we would like to be able to have limits  is in the panel materials, Plastic Sheets could never be greater than 4’x8’ sheet.
                We currently have no way to create a max “ Laenge “ value with these panels or the extrusions, and have these work with in the library
                Is there a “ simple “ way that you would solve this problem ? ( I know nothing is simple….haha

**Comment by eheyder** [comment]
Additional information from Kevin:
I have attached a ST9 part file to this e-mail. The name of this part is  
Polycarbonate 6mm, clear_9000402-2438.4mm x 1219.2mm_-0_0mm.par    it is currently exactly  how we would like to see it named.
“Polycarbonate 6mm, clear_9000402-2438.4mm x 1219.2mm_-0_0mm.par”     The red numerals are for the MONTAGE dimension in the variable table, this is for a feature to add a preset length or width to the panel depending on the style of frame we are using. So we would like to leave that alone.
In the Variable table the L_x variable is the LAENGE_X dimension and is always placed first, this can never be longer than 8’0” or 96” or as listed in the limits ( 2438.4mm )
In the Variable table the B_x variable is the BREITE_X dimension and is always placed second, this can never be longer than 4’0” or 48” or as listed in the limits ( 1219.2mm )
                We will try to use this as our example and base all our current and future panels off of this when we get this one right.
[Polycarbonate 6mm, clear_9000402_2438.4mm_x1219.2mm_-0_0mm.par.txt](https://github.com/KeyboardLogic/KeyboardLogic.SolidEdge.AddIn.ItemCatalog/files/2342366/Polycarbonate.6mm.clear_9000402_2438.4mm_x1219.2mm_-0_0mm.par.txt)

**Comment by eheyder** [comment]
I have attached the  example from the e-mail sent to you on  09/06/2018. The part name appears ( in the mother folder ) as  
Plastic 4mm, black sim. to RAL9017 904830_0mm_0mm.par.
After we have selected that part from the catalog and added the new Example : L =1400 and the B=1016  variables the part will appear as its new name
Plastic 4mm, black sim. to RAL9017 904830_0mm_0mm-1400.0mm_1016.0mm.par.
                I hope this helps to clear this up a bit. Sorry for the bad explanation…
